UnitTests/TransportTests.cpp

```diff
@@ -18,6 +19,8 @@
 
 BOOST_AUTO_TEST_SUITE(TransportTests)
 
+using Traits = IPC::UnitTest::Mocks::NullTimeoutTraits;

```

Comment:
>Traits [](start = 6, length = 6)

I'd name it NullTimeoutTraits, or just used 'using IPC::UnitTest::Mocks::NullTimeoutTraits'